### PR TITLE
Translations ->HTML5

### PR DESCRIPTION
**Pull request by N3tt3rm4n**
Did I mention, I am crazy about HTML5? :grinning:
Well, I'm also crazy about tabs :smile:

I needed some work where I don't need to think 'cause I had headache ^^

### CODE REVIEW

**Comment by Super-Baleine** [comment]
Hey,
Please, solve conflicts before anything else. x)

**Comment by N3tt3rm4n** [comment]
Ups, I see there are some conflicts...
At the German version the new translation for "openSourceTxt" is `"Muonium ist vollständig Opensource-Software. Du kannst wann immer Du willst den Code auf <a href=\"https://github.com/muonium\">GitHub</a> sehen, prüfen und verbessern"`

**Comment by N3tt3rm4n** [comment]
Ok I'll change it quickly

**Comment by Super-Baleine** [comment]
Yea, do it, and I'll consider your PR! 😃

**Comment by N3tt3rm4n** [comment]
Updated

File: core/de.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "back": "Zurück zu Muonium",
         "favorites": "Favoriten",
         "login": "Login",
-        "notready": "Geschlossene Beta, willkommen zum Königreich der Wale",
+        "notready": "Private Beta, willkommen im Königreich der Wale",
```
Comment (Super-Baleine) [inline core/de.json:8]: Why do you use an English word?

Comment (N3tt3rm4n) [reply core/de.json:8]: "Private Beta" is also German, and "geschlossene Beta" would have meant something like closed Beta, that's why I changed it

Comment (Super-Baleine) [reply core/de.json:8]: Yea okay... I just had some troubles with translations... Then perhaps there're some things to fix, I don't know.

Comment (N3tt3rm4n) [reply core/de.json:8]: I'll have a look at it

**Pull request merged**